packages/tools/devtools/devtools-core/package.json

```diff
@@ -119,6 +119,9 @@
 	},
 	"typeValidation": {
 		"broken": {
+			"EnumDeclaration_DevtoolsFeature": {
+				"backCompat": false
+			},
```

Comment:
This is already covered below in line 129, can be removed.

Reply:
removed.